webapp-django/crashstats/crashstats/static/crashstats/css/base.less

```diff
@@ -1,5 +1,254 @@
-@import "reset.less";
-@import "variables.less";
-@import "mixins.less";
-@import "forms.less";
-@import "tables.less";
+/*
+Copyright (c) 2009, Yahoo! Inc. All rights reserved.
+Code licensed under the BSD License:
+http://developer.yahoo.net/yui/license.txt
+version: 3.0.0
+build: 1549
+*/
+
+@import "base/reset.less";
+@import "base/variables.less";
+@import "base/mixins.less";
+@import "base/forms.less";
+@import "base/tables.less";
+@import "base/typography.less";
+@import "base/layout.less";
+
+/* Layout elements */
+@import "components/nav.less";
+@import "components/footer.less";
+
+/* Common components */
```

Comment:
Note: the dream is to eventually move these component `import`s to the pages that actually require them (where possible). For now, they are all included here as a temporary measure.